### PR TITLE
Foundry Update

### PR DESCRIPTION
**Pull request by Black0305**
Unification:
- Added Blocks/Gears/Rods/Plates

Foundry:
- Gears in Metal Caster 6 Ingots
- Plates in Metal Caster 1,5 Ingots
- Rods in Metal Caster 1 Ingot
- Removed a lot of auto-generated recipes
Same amounts apply to the Casting Tables

### CODE REVIEW

**Comment by Black0305** [comment]
For now the Metal Caster recipes are limited to ingots which are smeltable in the Melting Crucible. If you want recipes for other liquid metals too, just let me know.

**Comment by Agrax** [comment]
I like it

File: scripts/foundry.zs [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import crafttweaker.liquid.ILiquidStack;
 
 mods.foundry.Melting.clear();
-//mods.foundry.Casting.clear(); Casting doesn't clear yet.  https://github.com/Shadows-of-Fire/Foundry/issues/30
+mods.foundry.Casting.clearRecipes();
+mods.foundry.Casting.clearMolds(); //Casting doesn't clear yet. https://github.com/Shadows-of-Fire/Foundry/issues/30 Fixed.
```
Comment (pyure) [inline scripts/foundry.zs]: Appreciate the comment

File: scripts/foundry.zs [hunk 2]
```diff
@@ -101,8 +111,39 @@ val tool_array = [
   "WroughtIron" 
 ] as string[];
 
-val ingot_casting_array = [
-  "Cobalt"
+val gear_array = [
```
Comment (pyure) [inline scripts/foundry.zs]: These arrays are kinda nutty, but there's no way to define real structs or classes.

**Pull request merged**